pyproject.toml

```diff
@@ -1,6 +1,130 @@
+[project]
+name = "tweakwcs"
+description = "A package for correcting alignment errors in WCS objects"
+requires-python = ">=3.8"
+authors = [
+    { name = "Mihai Cara", email = "help@stsci.edu" },
+]
+dependencies = [
+    "numpy",
+    "astropy",
+    'astropy>=5.0.4',
+    'gwcs>=0.14.0',
+    'stsci.stimage',
+    'stsci.imagestats',
+    'spherical_geometry>=1.2.20',
+    'packaging>=19.0',
+]
+dynamic = [
+    "version",
+]
+classifiers = [
+    'Intended Audience :: Science/Research',
+    'License :: OSI Approved :: BSD License',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python',
+    'Topic :: Scientific/Engineering :: Astronomy',
+    'Topic :: Software Development :: Libraries :: Python Modules',
+    'Development Status :: 4 - Beta',
+]
+
+[project.readme]
+file = "README.rst"
+content-type = "text/x-rst"
+
+[project.license]
+file = "LICENSE.txt"
+content-type = "text/plain"
+
+[project.urls]
+Homepage = "https://github.com/spacetelescope/tweakwcs"
+"Bug Tracker" = "https://github.com/spacetelescope/tweakwcs/issues"
+Documentation = "http://spacetelescope.github.io/tweakwcs/"
+"Source Code" = "https://github.com/spacetelescope/tweakwcs"
+
+[project.optional-dependencies]
+test = [
+    "pytest",
+    "pytest-cov",
+    "coverage",
+    "scipy",
+]
+docs = [
+    "tomli; python_version<'3.11'",
+    'numpydoc',
+    'sphinx',
+    'sphinx-automodapi',
+    'sphinx-rtd-theme',
+    'stsci-rtd-theme',
+    'sphinx-astropy',
+    'graphviz',
+]
+
 [build-system]
 requires = [
-    "setuptools>=38.2.5",
-    "setuptools_scm",
+    "setuptools>=61.2",
+    "setuptools_scm[toml]>=3.6",
     "wheel",
 ]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+zip-safe = false
+include-package-data = true
+license-files = ["LICENSE.rst"]
+
+[tool.setuptools.packages.find]
+namespaces = false
+
+[tool.setuptools.package-data]
+"*" = [
+    'README.rst',
+    'LICENSE.txt',
+    'CHANGELOG.rst',
+    '*.fits',
+    '*.txt',
+    '*.inc',
+    '*.cfg',
+    '*.csv',
+    '*.yaml',
+    '*.json',
```

Comment:
Should be more specific on these ones. You don't want to bundle accidental data.